### ISSUE TITLE
Allow more consistent degrees=None in RandomAffine constructor

### ISSUE THREAD

**Issue by vadimkantorov** [pytorch/vision#5241]
### 🐛 Describe the bug

All other arguments allow for setting None to desactivate, except degrees. For the cases of when only some of these arguments are set, allowing same values for desactivation is a bit better.

### Versions

N/A

**Comment by datumbox** [comment]
Thanks for the proposal. This is probably a feature request/enhancement rather than a bug.

cc @vfdev-5 as I know you've been looking into affine recently.

**Comment by vfdev-5** [comment]
Basically, the proposal is about 
```diff
- RandomAffine(0, translate=(0.2, 0.3))
+ RandomAffine(translate=(0.2, 0.3))
```

It is possible to implement that, while thinking about raising an error for `RandomAffine()`.

**Comment by vadimkantorov** [comment]
I would even say that `RandomAffine()` should not raise an error and just be equivalent to Identity

**Comment by vfdev-5** [comment]
`RandomAffine() == Identity` I'm not sure about such API.

@vadimkantorov would you like to send a PR with this feature ?

**Comment by vadimkantorov** [comment]
> why to use `RandomAffine()` as indentity.

I think no-one would use RandomAffine() in place of identity, but it may simplify error handling in user code that just pass through the arguments to RandomAffine (e.g. optional affine transform as part of more complex augmentation pipeline), and then it's better to have simpler error handling and ideally avoid it altogether (just like it's meaningful to sometimes support zero batch size / empty tensors etc, it makes handling of certain edge cases much simpler in the user code)

**Comment by vfdev-5** [comment]
For me it is a bit different. I'd compare that to configuring `Conv2d()` without any params and say that it should behave as Identity in this case. A use-case where `RandomAffine` configuration can be a sort of input (as you talk about zero batch tensors etc) is when user does some search on dataaug and reconfigure `RandomAffine` differently during the search and would like to provide an empty config and have everything working. In general case, I'd say we have to raise an error on empty configuration.

**Comment by vadimkantorov** [comment]
Yeah, my argument was about your second paragraph. Imagine a complex augmentation pipeline that reads affine params from config (or constructor arguments) - this happens in practice quite a lot. In this complex pipeline RandomAffine is just one component. For this setting, disabling Affine transform by letting the final user to set all affine params None / zero is a valid usecase, and no error (identity stub) frees the user from error handling or special-casing this behavior. e.g. Dropout with p = 0 or p = 1 is accepted, in the former case Identity is performed, but of course torchvision can deviate from torch core designs.

I also don't mind allowing plain 0 (in place of None) to specify Identity behavior (although probably if all-zeros is accepted, all-Nones could be accepted as well).

**Comment by zy1git** [comment]
After discussion, we don’t think this is a current priority for TorchVision. so I’m going to close this issue for now.
